### DISCUSSION TITLE
How to couple subapp and masterapp with different timescales?

### DISCUSSION THREAD

**Discussion by xiaojbing** [idaholab/moose#26496] [Q&A General]
for example, the timescale of subapp is 0.1s (time 1 in MOOSE has the unit 0.1s), but the main app is slower (time 1 in MOOSE has the unit 0.01s). The two apps are transient apps. How to couple them with a multiapp system?

**Comment by GiudGiud** [comment]
Hello

See these slides for the various time stepping options during coupling
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial02_multiapps/presentation/index.html#/2/10

Guillaume

**Reply by xiaojbing** [reply]
Thank you, but I think you mean using `sub_cycling `to let the main app and the sub app have different time steps during the iteration. And what I'm asking is how to couple the main app and the sub app when they represent physics that are not as fast as each other (they have different time unit).

**Reply by GiudGiud** [reply]
Should you be solving one to steady state every time you want to couple them at a timestep of the other then?

Use a fullsolvemultiapp instead of a transientmultiapp for that
